Firmware/Marlin_main.cpp

```diff
@@ -7819,6 +7831,10 @@ SERIAL_PROTOCOLPGM("\n\n");
 	{
 		eeprom_update_byte(&EEPROM_Sheets_base->s[iSel].pinda_temp, iPindaC);
 	}
+	if (bHasIsActive)
+	{
+		if(bIsActive) eeprom_update_byte(&EEPROM_Sheets_base->active_sheet, iSel);
+	}
```

Comment:
This can be optimised into one `if` statement

`if (bHasIsActive && bIsActive) eeprom_update_byte(&EEPROM_Sheets_base->active_sheet, iSel);`

I don't trust the compiler to do this optimisation everytime.